content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md

```diff
@@ -0,0 +1,75 @@
+date: 2024-03-26
+images:
+  - /blog-images/2024/march24events.png
+post: 
+    title: "SCaLE 21x, Hackathon, AlmaLinux Day, and CloudFest!"
+    image: /blog-images/2024/march24events.png
+---
+
+
+March has been a whirlwind and in the last two weeks our event teams visited two sides of the world! In March we saw both Pasedena, California and Rust, Germany! Let's talk about what we loved about each event. 
+
+## SCaLE 21x
+
+The 21st iteration of the Southern California Linux Expo (SCaLE) kicked off on March 14th, and we were fortunate enough to have a booth staffed by a member of the AlmaLinux OS Foundation. We got to see a ton of our community, and were especially excited to connect with a number of folks who wanted to contribute to AlmaLinux.
+
+This continues to be one of our most beloved shows. It's so nice to see it growing and returning to its pre-pandemic size. We are so grateful for the continued outpouring of support from our Linux community there.
+
+## CloudFest Hackathon
+
+As SCaLE was hitting its stride, 5300 miles away in Rust, Germany, our other event team was arriving and getting ready to start hacking at the CloudFest Hackathon. As one of the [Hackathon projects](https://hackathon.cloudfest.com/agenda/), a group of people joined us to help [improve the security of infrastructure globally by easing OS upgrades](https://hackathon.cloudfest.com/project/securing-more-infrastructure-by-easing-os-upgrades/). We were delighted with what we accomplished there, and we were awarded 3rd place in both the "Future of the Web" and "Web Impact" categories.
+
+We chose three primary goals for the Hackathon. Below I've included a bit more about our goals and the results of the work we did!
+
+### What ideas did we have, and what were the results?
+
+-   Deep Dive into User Space: Analyze commonly used software and CMSs to pinpoint potential upgrade obstacles, ensuring smoother transitions
+   -   We tested 3 different real-world scenarios and reported bugs against each scenario.
+-   Focus on Industry Drivers: Identify key elements crucial to data consistency and other industry needs to facilitate necessary upgrades.
+   -   We identified and submitted patches for a number of drivers that could be easily and safely re-added to the kernel, allowing users a chance to continue using unsupported hardware (keep an eye out for the AlmaLinux 8 and 9 betas next month!)
+-   Expand ELevate Support: Extend the capabilities for in-place upgrades across enterprise Linux distributions, including critical updates from CentOS 6 to 7.
+   -   We completed enough research to determine that it's possible and look forward to trying to implement the research over the coming months.
+
+Here are the slides from our [project reporting in PDF format](https://www.dropbox.com/scl/fi/qkokxhum5xsy2pade8qb5/EasingOSUpgrades-cfhack-2024.pdf?rlkey=lb7otq7euorbyj8s2ozo5czo3&dl=0), in case folks would like to look at the details related to our work.
```

Comment:
Thoughts on uploading this so we serve it as a static asset/it doesn't go poof `N` years from now? 

PTSD from the missing Jack video.

Reply:
good. catch.